editoast/src/views/timetable/similar_trains.rs

```diff
@@ -1129,4 +1129,196 @@ mod tests {
             }
         );
     }
+
+    #[rstest]
+    async fn several_similar_trains() {
+        let db_pool = DbConnectionPoolV2::for_tests();
+        let mut core = MockingClient::new();
+        core.stub("/pathfinding/blocks")
+            .method(reqwest::Method::POST)
+            .response(StatusCode::OK)
+            .json(PathfindingResult::Success(pathfinding_result_success()))
+            .json(PathfindingResult::Success(pathfinding_result_success()))
+            .json(PathfindingResult::Success(pathfinding_result_success()))
+            .json(PathfindingResult::Success(pathfinding_result_success()))
+            .json(PathfindingResult::Success(pathfinding_result_success()))
```

Comment:
Nice trick on not mocking pathfinding responses! 😁 (maybe a line of documentation on `pathfinding_result°success` to explain why we don't actually need the path result for the test would be nice?)